testing/utilities.py

```diff
@@ -523,3 +548,35 @@ def test_set_scenario(self):
         scenario_set = requests.get('{0}/scenario'.format(self.url)).json()
         self.assertEqual(scenario, scenario_set)
         requests.put('{0}/scenario'.format(self.url), data=scenario_current)
+
+    def test_partial_results_inner(self):
+        '''Test getting results for start time after and final time before.
+
+        '''
+
+        requests.put('{0}/initialize'.format(self.url), data={'start_time':0, 'warmup_period':0})
+        requests.put('{0}/step'.format(self.url), data={'step':self.step_ref})
+        measurements = requests.get('{0}/measurements'.format(self.url)).json()
+        y = requests.post('{0}/advance'.format(self.url), data=dict()).json()
+        res_inner = requests.put('{0}/results'.format(self.url), data={'point_name':measurements.keys()[0], \
+                                                                 'start_time':self.step_ref*0.25, \
+                                                                 'final_time':self.step_ref*0.75}).json()
+        df = pd.DataFrame.from_dict(res_inner).set_index('time')
+        ref_filepath = os.path.join(get_root_path(), 'testing', 'references', self.name, 'partial_results_inner.csv')
+        self.compare_ref_timeseries_df(df, ref_filepath)
+
+    def test_partial_results_outer(self):
+        '''Test getting results for start time before and final time after.
+
+        '''
+
+        requests.put('{0}/initialize'.format(self.url), data={'start_time':0, 'warmup_period':0})
+        requests.put('{0}/step'.format(self.url), data={'step':self.step_ref})
+        measurements = requests.get('{0}/measurements'.format(self.url)).json()
+        y = requests.post('{0}/advance'.format(self.url), data=dict()).json()
```

Comment:
Variable `y` is not used after assignment. I'd suggest to remove or to change by `_`

Reply:
Addressed in https://github.com/ibpsa/project1-boptest/pull/268/commits/ba2364682a3d59e684cfb42916f1e23050b8c4c0.